### PR TITLE
#59 add bootstrap logging to Improve the HTML logger

### PR DESCRIPTION
**Pull request by JeffreyZacher**
after working on it I don't think it is possible to have a jumbotron with the assert counts in it. VSTestContext doesn't have that information, and Nunit test context doesn't count how many asserts are ran.
 So I removed it

### CODE REVIEW

File: Framework/Utilities/Logging/HtmlFileLogger.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ public class HtmlFileLogger : FileLogger, IDisposable
...
+        private const string SCRIPTANDCSSTAGS = "<style>.modal-dialog{max-width: fit-content}</style><script>$(function (){$('.pop').on('click', function (e){$('.imagepreview').attr('src', $(this).find('img').attr('src'));$('#imagemodal').modal('show');});});</script><script>$(function (){$('.pop2').on('click', function (){$('.imagepreview').attr('src', $(this).attr('src'));$('#imagemodal').modal('show');});});</script><script>$(function (){$('.dropdown-item').on('click', function (e){$(this).attr('class', function (i, old){return old=='dropdown-item' ? 'dropdown-item bg-secondary' :'dropdown-item';});var temp=$(this).data('name');$('[data-logtype=\\\'' + temp + '\\\']').toggleClass('show');e.stopPropagation();});});</script>";
```
Comment (TroyWalshProf) [inline Framework/Utilities/Logging/HtmlFileLogger.cs:33]: Strongly suggest changing 
`<style>.modal-dialog{max-width: fit-content}</style>`
to
<style>.modal-dialog{max-width: fit-content} p{white-space: pre-wrap}</style>
This way line feeds are respected

File: Framework/Utilities/Logging/HtmlFileLogger.cs [hunk 2]
```diff
@@ -136,24 +143,24 @@ private string GetTextWithColorFlag(MessageType type)
             switch (type)
             {
                 case MessageType.VERBOSE:
-                    return "<p style =\"color:purple\">";
+                    return "text-secondary";
                 case MessageType.ACTION:
```
Comment (TroyWalshProf) [inline Framework/Utilities/Logging/HtmlFileLogger.cs:147]: How would you feel about?
Verbose -> bg-secondary
Action -> text-info
Step -> text-primary
Information-> text-secondary
*Just looks a little more natural, at least to me
See - 
image

File: Framework/Utilities/Logging/HtmlFileLogger.cs [hunk 3]
```diff
@@ -23,10 +23,24 @@ public class HtmlFileLogger : FileLogger, IDisposable
...
+
+        /// <summary>
+        /// Contains the CSS tags, and the JQuery Scripts
+        /// </summary>
+        private const string SCRIPTANDCSSTAGS = "<style>.modal-dialog{max-width: fit-content} p{white-space: pre-wrap}</style><script>$(function (){$('.pop').on('click', function (e){$('.imagepreview').attr('src', $(this).find('img').attr('src'));$('#imagemodal').modal('show');});});</script><script>$(function (){$('.pop2').on('click', function (){$('.imagepreview').attr('src', $(this).attr('src'));$('#imagemodal').modal('show');});});</script><script>$(function (){$('.dropdown-item').on('click', function (e){$(this).attr('class', function (i, old){return old=='dropdown-item' ? 'dropdown-item bg-secondary' :'dropdown-item';});var temp=$(this).data('name');$('[data-logtype=\\\'' + temp + '\\\']').toggleClass('show');e.stopPropagation();});});</script><script>$(function (){$(document).ready(function(){$('#Header').append($('title').text())})});</script>";
```
Comment (TroyWalshProf) [inline Framework/Utilities/Logging/HtmlFileLogger.cs:33]: Suggest changing 
$('#Header').append($('title').text()
to 
$('#Header').append('&nbsp;' + $('title').text()
looks a little better

Comment (JeffreyZacher) [reply Framework/Utilities/Logging/HtmlFileLogger.cs:33]: the script will look better, or the the results looks better?

File: Framework/Utilities/Logging/HtmlFileLogger.cs (hunk 2 above)
Comment (JeffreyZacher) [reply Framework/Utilities/Logging/HtmlFileLogger.cs:147]: I like it, did you want the background color in? I can just do a style tag to get a different color text pretty easy

File: Framework/Utilities/Logging/HtmlFileLogger.cs (hunk 3 above)
Comment (TroyWalshProf) [reply Framework/Utilities/Logging/HtmlFileLogger.cs:33]: The report will look better

**Pull request merged**